README.md

```diff
@@ -49,4 +49,14 @@ juju crashdump [-h] [-d] [-m MODEL] [-f MAX_FILE_SIZE] [-b BUG]
 <dd>Enable the addon with the given name</dd>
 <dt>--addons-file ADDONS_FILE</dt>
 <dd>Use this file for addon definitions</dd>
+<dt>--as-root</dt>
+<dd>Collect logs as root, may contain passwords etc. Addons with local commands will only run if this flag is enabled.</dd>
```

Comment:
local to the machine running crashdump, or local to the juju machine?

Reply:
Local is the machine running crashdump, the juju machines are the `remotes`.

Reply:
I put a clarification in the addon example below.